### DISCUSSION TITLE
Counting the same relation multiple times in one query

### DISCUSSION THREAD

**Discussion by jamalmoir** [prisma/prisma#15387] [Q&A] [answered]
Is there a way to do something like this:

```
db.recipe.findMany({
  include: {
    _count: {
      select: { 
        "fruit_count": { where: {category: "Fruit"}, field: "ingredients"} // count of fruit recipes labelled as "fruit_count"
        "veg_count": { where: {category: "Vegetable"}, field: "ingredients"} // count of vegetable recipes labelled as "vege_count"
      }
    }
  }
})
```

It would be very useful to build quick stat queries.

Cheers!

**Comment by austincrim** [comment, marked as answer]
Hey @jamalmoir 👋,

This is not currently possible with Prisma but sounds like a good idea for a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=).

In the meantime, you can use [$queryRaw](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw) to accomplish the same thing.

**Reply by jamalmoir** [reply]
Great, thanks! I'll use raw queries for now and make a feature request!

**Reply by osylka-godaddy** [reply]
is feature request was created ?

**Reply by jamalmoir** [reply]
Yes, it was

**Reply by rohanrajpal** [reply]
Linking it here for anyone who wants to +1 https://github.com/prisma/prisma/issues/15423

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
